docker/development.yml

```diff
@@ -12,6 +12,24 @@ version: '3'
 services:
   # API Gateway
   nginx:
+    build:
+      context: ../src/web
+      dockerfile: Dockerfile
```

Comment:
Because we never change the name of the `Dockerfile`, I think this is unnecessary in each of the definitions in this file, see https://docs.docker.com/compose/compose-file/compose-file-v3/#context.